### PR TITLE
Convert repository from glide to go module.

### PR DESCRIPTION
**Pull request by Billy99**
This PR converts the repository from glide to gomod.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module github.com/openshift/app-netutil
+
+go 1.15
+
+require (
+	github.com/containernetworking/cni v0.7.1
+	github.com/go-logfmt/logfmt v0.0.0-20200104052239-3be5f6aae7db // indirect
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/intel/userspace-cni-network-plugin v0.0.0-20200416024655-673eeb0d731f
+	gopkg.in/intel/multus-cni.v3 v3.0.0-20200515123326-4eac660359f2
```
Comment (zenghui-shi) [inline go.mod:10]: @Billy99 did you pin multus/containernetworking to a certain version? 

I ran it (`go mod tidy`) locally and got this go.mod:
```
require (
        github.com/containernetworking/cni v0.8.0
        github.com/go-logfmt/logfmt v0.5.0 // indirect
        github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
        github.com/intel/userspace-cni-network-plugin v0.0.0-20200416024655-673eeb0d731f
        gopkg.in/intel/multus-cni.v3 v3.4.2
)
```

Comment (Billy99) [reply go.mod:10]: I was not trying to pin to a certain version. I did run `go mod tidy` (just looked at my history to confirm). I'll run again tomorrow and see what I come up with. Thanks for checking.

Comment (zenghui-shi) [reply go.mod:10]: Not a big issue, I was just trying to see if it's because we have issue using latest multus or other vendor pkgs.

Comment (zenghui-shi) [reply go.mod:10]: btw, when I ran the go mod tidy, I first removed the glide files, otherwise it will import from glide when generating go mod.

Comment (Billy99) [reply go.mod:10]: That's it, I left the glide files there when I did the initial import.

Comment (zenghui-shi) [reply go.mod:10]: Since this doesn't have any functional impact, I will merge this PR.
We will update vendor pkgs to latest in another PR.

**Pull request merged**